model/version.go

```diff
@@ -129,6 +129,50 @@ func (v *Version) ActivateAndSetBuildVariants() error {
 	)
 }
 
+// GetDisplayStatus considers both child patch statuses and
+// aborted status, and returns an overall status.
+func (v *Version) GetDisplayStatus() (string, error) {
```

Comment:
This does ultimately return a version status -- we convert it to patch to get the cumulative but then we convert it back. I can add a comment to that effect.

Reply:
Okay _actually_ since this is only used by the resolver which relies on the patch status (as referenced by minna) i'll just have this return the patch status.